main.tf

```diff
@@ -112,7 +112,7 @@ module "main" {
     },
     {
       name  = "MONGO_URI"
-      value = "mongodb+srv://${data.aws_ssm_parameter.db_user.value}:${data.aws_ssm_parameter.db_password.value}@${data.aws_ssm_parameter.db_host.value}/fightpandemics?retryWrites=true&w=majority"
+      value = "mongodb+srv://${data.aws_ssm_parameter.db_user.value}:${data.aws_ssm_parameter.db_password.value}@${data.aws_ssm_parameter.db_host.value}/${var.env_name}?retryWrites=true&w=majority"
```

Comment:
@Zaydme was this supposed to be temporary - to need a separate db from other review environments?

Or if there was an issue you encountered what was it. There are a few reasons we want to use the same db /fightpandemics for all of review. Maybe usage limits, not needing reviewers to keep creating new user (although maybe that would be a good thing?).  Can't quite recall the full lists of reasons, @mannykary?

Reply:
Yeah, it had to do with the indexes. The database is shared by all review builds so the indexes would be created by this build, only to be dropped by other builds. I suggested creating a new database to work around this issue, but I would recommend reverting this after this has been tested by QA and before merging to staging